walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -24,7 +24,11 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
+     *
+     */
+    private static final long serialVersionUID = -3908980907790412313L;
```

Comment:
Another thing is: what coding style guide are you following? Usually a "static final immutable" object is a *constant*, and we usually using all captital letters:

private static final long SERIAL_VERSION_UID = -....

Reply:
See https://google.github.io/styleguide/javaguide.html